.github/workflows/cluster.yaml

```diff
@@ -0,0 +1,60 @@
+name: CI
```

Comment:
The Slurm CI can be placed in `python.yaml`. In this way, it can be achieved that the Slurm test is performed after the `python lint`.
You can add an extra `matrix.module`, refer to `kubernetes-juicefs` in `python.yaml`.